.gitignore

```diff
@@ -8,3 +8,6 @@ output
 *.DS_Store
 tags
 spec/examples.txt
+node_modules/
+.cache/
+public
```

Comment:
Rails uses the public folder for static assets, don't think we use it here though so should not be a problem. Worth keeping in mind though if you see some odd behaviour with those @MymmiJ

Reply:
Cheers, it doesn't look used but I'll keep an eye on it